src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx

```diff
@@ -0,0 +1,227 @@
+---
+title: Alerts and Applied Intelligence notification integrations
+tags:
+  - Alerts and Applied Intelligence
+  - Applied intelligence
+  - Incident Intelligence
+  - Destinations
+metaDescription: "Alerts and Applied Intelligence notification integrations are specific services and platforms you can use to send notifications from New Relic."
+---
+
+<Callout title="Early access">
+The features described here are early access.
+
+For more information on related features, see our docs on [Alerts notification channels](/docs/alerts-applied-intelligence/new-relic-alerts/alert-notifications/notification-channels-control-where-send-alerts/), [Incident Intelligence destinations](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/incident-intelligence-destination-examples/), and [Proactive Detection notifications](https://docs.newrelic.com/docs/alerts-applied-intelligence/applied-intelligence/proactive-detection/proactive-detection-applied-intelligence/#set-up).
+</Callout>
+
+Alerts and Applied Intelligence notification integrations are specific services and platforms you can use to send notifications from New Relic.
+
+## Notification integration detail [#detail]
+
+Read about more about each of our specific notification integrations.
```

Comment:
```suggestion
Read more about each of our specific notification integrations.
```

Reply:
Thanks! Sometimes I swap words and letters around like that. Weirdly, it's tough to spot them later :)